Makefile

```diff
@@ -3,6 +3,6 @@ dev-build:
 build:
 	CGO_ENABLED=0 GOOS=linux go build -a -installsuffix cgo -o policy_asserter .
 dockerize: build
-	docker build . --tag $DOCKER_REPO/sslpolicy-asserter:1.1
+	docker build . --tag $DOCKER_REPO/sslpolicy-asserter:1.2
 publish: dockerize
```

Comment:
do we need Docker related steps here and have a cloudbuild.yaml?

Reply:
It will be fixed after the https://github.com/commercetools/gcp-ssl-policy-asserter/pull/5 merge.

Reply:
As mentioned above, this will be fixed by https://github.com/commercetools/gcp-ssl-policy-asserter/pull/5